tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs

```diff
@@ -91,9 +91,9 @@ private static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command, Int
                     dbType = DbType.Sqlite;
                     return true;
                 default:
-                    integrationId = null;
-                    dbType = null;
-                    return false;
+                    integrationId = IntegrationId.AdoNet;
+                    dbType = commandTypeFullName.Substring(commandTypeFullName.LastIndexOf(".") + 1).ToLowerInvariant();
+                    return true;
```

Comment:
I remember for callsite we were skipping some known command types that we don't want to instrument, do you remember @lucaspimentel ? Internals for entity framework perhaps? We should add that logic here as well.

Reply:
maybe these: https://github.com/DataDog/dd-trace-dotnet/blob/9d9bb060d819925fd57baecfe5b1cd168f6e6706/src/Datadog.Trace.ClrProfiler.Managed/ScopeFactory.cs#L209-L215

Reply:
Added both in 9b85d63

Reply:
@tonyredondo: We ignored `InterceptableDbCommand` and `ProfiledDbCommand`, and we tried to get a name from the comment type. The logic was here, which I removed recently (#2008) because we didn't use it anymore 😅 
https://github.com/DataDog/dd-trace-dotnet/blob/af4451b193e287797065e76e10527e098358d8bf/tracer/src/Datadog.Trace/ClrProfiler/ScopeFactory.cs#L199-L237

Reply:
See also some test cases here
https://github.com/DataDog/dd-trace-dotnet/blob/af4451b193e287797065e76e10527e098358d8bf/tracer/test/Datadog.Trace.ClrProfiler.Managed.Tests/ScopeDBFactoryTests.cs#L96-L109

Reply:
I've reverted the logic and test in 3b84f2d